### ISSUE TITLE
error publishing new release

### ISSUE THREAD

**Issue by codeboten** [open-telemetry/opentelemetry-collector-contrib#18471]
### Describe the issue you're reporting

The `publish-stable` job failed because it couldn't find `ghr`:

https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/4128527733/jobs/7133733613

**Comment by jpkrohling** [comment]
What's `ghr`? Shouldn't we be using `gh` instead, which is the official GitHub CLI tool, included in the GitHub CI images by default?

**Comment by codeboten** [comment]
I don't think we need ghr, since we're not actually uploading any binaries as part of the release of this repo. I've added a PR to use gh instead. PTAL
